cloudformation/cfncluster.cfn.json

```diff
@@ -1976,6 +1976,18 @@
         }
       }
     },
+    "NewInstanceProfile": {
```

Comment:
Instead of creating a new instance profile why don't you use the existing `RootInstanceProfile` and use an if statement to reference the role. https://github.com/awslabs/cfncluster/blob/develop/cloudformation/cfncluster.cfn.json#L1853

Reply:
Sure, that would also work.